packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1097,6 +1097,7 @@ export interface CallWithChatAdapterManagement {
     fetchInitialData(): Promise<void>;
     // @beta
     holdCall: () => Promise<void>;
+    isChatAdapterInitialized(): boolean;
```

Comment:
The concept that the callWithChat has two internal adapters isn't exposed in the public API, keep this name agnostic of the internal implementation, suggestion:
```suggestion
    isChatInitialized(): boolean;
```

Reply:
We also don't have getters in this interface, any property that changes over time is in the `state` object, move `isChatInitialized` to `CallWithChatClientState`